app/forms/legal_aid_applications/has_evidence_of_benefit_form.rb

```diff
@@ -9,7 +9,7 @@ class HasEvidenceOfBenefitForm < BaseForm
   private
 
     def passporting_benefit_error
-      ".shared.forms.received_benefit_confirmation.form.providers.received_benefit_confirmations.#{model.passporting_benefit}"
+      ".shared.forms.received_benefit_confirmation.form.providers.received_benefit_confirmations.#{model.passporting_benefit.squish.downcase.tr(' ', '_')}"
```

Comment:
this is just an extra I assume. not to do with rails bump?

Reply:
This code was a safety check and a bit weird, it would send the checkbox value, e.g. `Universal credit` to find an error.
So the string interpolation would try and find `.shared.forms.received_benefit_confirmation.form.providers.received_benefit_confirmations.Universal Credit` and fail

This part of the i18n update now, correctly, raised an error!